### DISCUSSION TITLE
[FEEDBACK] - Interoperability: must support JSON as a minimum

### DISCUSSION THREAD

**Discussion by HealthOne-Wayne** [tewhatuora/api-standards#115] [General]
### Summary

What is the reason for choosing JSON at minimum, and not XML? And why one or the other as a minimum as FHIR supports both.

Perhaps minimum of JSON could apply for new APIs as there are existing XML only FHIR APIs

### Link to standards item

https://apistandards.digital.health.nz/fhir-api-standard/Standards/Interoperability

### Which area of the standards does this apply to?

- [ ] Part A - API Concepts
- [ ] Part B - API Security
- [X] Part C - API Design and Development
- [ ] Part D - FHIR
- [ ] Community guidelines

**Comment by swithinfoote** [comment]
Hi @HealthOne-Wayne - we feel that this item is a candidate for more discussion so have moved to the discussions area.

**Comment by mathusmith** [comment]
While FHIR supports both JSON and XML, Te Whatu Ora is firmly of the view of standardising on one format for structured data exchange in REST APIs. The ubiquity of support and use of JSON for REST APIs makes it by far the preference. 
The ease of converting between XML and JSON means that this should not be a hindrance in interoperating with XML encoded data sources be they FHIR or HL7V2.

**Comment by stueynz** [comment]
> Perhaps minimum of JSON could apply for new APIs as there are existing XML only FHIR APIs

I agree with this sentiment.   I would hate to see someone saying that we MUST (because of the new API Standard) update existing XML APIs to be JSON.

**Comment by mathusmith** [comment]
Reply to @stueynz the intent of the standard is to be forward looking rather than backward. Going forward, JSON is what Te Whatu Ora wanting to standardise on. So far as the need to remediate existing XML based APIs, this will form part of the approach for co-existence in which HL7v2x/v3 will also feature prominently.
